eng/tools/sdk-suppressions/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@azure-tools/sdk-suppressions",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/index.js",
+  "version": "1.0.0",
+  "bin": {
+    "get-sdk-suppressions-label": "cmd/sdk-suppressions-label.js"
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "start": "npm run build && node cmd/sdk-suppressions-label.js"
+  },
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "dependencies": {
+    "ajv": "^6.12.6",
+    "js-yaml": "^3.14.0",
+    "lodash": "^4.17.20"
+  },
+  "devDependencies": {
```

Comment:
Add `@types/node` and `typescript` devDeps matching other tools:

https://github.com/Azure/azure-rest-api-specs/blob/1e620cfbf7df188acef4d6b4a8752aa3aa82fa02/eng/tools/typespec-validation/package.json#L16-L18

Reply:
add unit test code